v03_pipeline/lib/reference_data/compare_globals.py

```diff
@@ -15,131 +19,123 @@
 logger = logging.getLogger(__name__)
 
 
+def struct_to_dict(struct: hl.Struct) -> dict:
```

Comment:
related comment: https://github.com/broadinstitute/seqr-loading-pipelines/pull/676/files#r1471678310

Reply:
ok I ended up just removing this function and using `dict()`!